### DISCUSSION TITLE
Extension Manager 0.3.1

### DISCUSSION THREAD

**Discussion by mjakeman** [mjakeman/extension-manager#207] [Announcements]
Minor update

Refer to Extension Manager 0.3 release notes:
https://github.com/mjakeman/extension-manager/releases/tag/v0.3.0

## What's Changed
* Updated Dutch translation by @Vistaus in https://github.com/mjakeman/extension-manager/pull/151
* Update Polish translation by @rafal06 in https://github.com/mjakeman/extension-manager/pull/152
* Updated Russian (very minor fixes) by @Danik2343 in https://github.com/mjakeman/extension-manager/pull/159
* Switch to GNOME 42 runtime by @mjakeman in https://github.com/mjakeman/extension-manager/pull/155
* Updated french translations by @GeoffreyCoulaud in https://github.com/mjakeman/extension-manager/pull/156
* Update German translation by @gabrbrand in https://github.com/mjakeman/extension-manager/pull/166
* Add Croatian language by @muzena in https://github.com/mjakeman/extension-manager/pull/160
* Fix translator credits for French translation by @mjakeman in https://github.com/mjakeman/extension-manager/pull/169
* po: Update Brazilian Portuguese translations by @douglasilvadev in https://github.com/mjakeman/extension-manager/pull/168
* po: Add Danish Translation by @exponentactivity in https://github.com/mjakeman/extension-manager/pull/149
* Update hu.po by @ViBE-HU in https://github.com/mjakeman/extension-manager/pull/170
* Set window title by @oscfdezdz in https://github.com/mjakeman/extension-manager/pull/176
* Update oc translation file by @Mejans in https://github.com/mjakeman/extension-manager/pull/177
* Fix special character escaping by @mjakeman in https://github.com/mjakeman/extension-manager/pull/175
* metainfo: Update for standard install location by @jbicha in https://github.com/mjakeman/extension-manager/pull/180
* Remove release notes popup dialog on startup by @mjakeman in https://github.com/mjakeman/extension-manager/pull/174
* metainfo: Update for standard install location, part two by @jbicha in https://github.com/mjakeman/extension-manager/pull/181
* Update oc.po: 100% by @Mejans in https://github.com/mjakeman/extension-manager/pull/188
* Refresh README by @mjakeman in https://github.com/mjakeman/extension-manager/pull/196
* po: Add Persian translation by @mjakeman in https://github.com/mjakeman/extension-manager/pull/205
* po: Add Persian (fa) Translation by @nimafanniasl in https://github.com/mjakeman/extension-manager/pull/186
* Po: Add Swedish Translation by @gullegugg in https://github.com/mjakeman/extension-manager/pull/195
* project: Release version 0.3.1 by @mjakeman in https://github.com/mjakeman/extension-manager/pull/206

## New Contributors
* @GeoffreyCoulaud made their first contribution in https://github.com/mjakeman/extension-manager/pull/156
* @gabrbrand made their first contribution in https://github.com/mjakeman/extension-manager/pull/166
* @muzena made their first contribution in https://github.com/mjakeman/extension-manager/pull/160
* @exponentactivity made their first contribution in https://github.com/mjakeman/extension-manager/pull/149
* @nimafanniasl made their first contribution in https://github.com/mjakeman/extension-manager/pull/186
* @gullegugg made their first contribution in https://github.com/mjakeman/extension-manager/pull/195

**Full Changelog**: https://github.com/mjakeman/extension-manager/compare/v0.3.0...v0.3.1

<hr /><em>This discussion was created from the release <a href='https://github.com/mjakeman/extension-manager/releases/tag/v0.3.1'>Extension Manager 0.3.1</a>.</em>

**Comment by fabd** [comment]
Great app, well done! Feels like it should be part of the OS.
